package.json

```diff
@@ -49,4 +49,4 @@
   "engines": {
     "node": ">=20.0"
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed this with the included .editorconfig file (last commit)